package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "solarized-light-syntax",
   "theme": "syntax",
-  "version": "0.23.0",
+  "version": "0.23.1",
```

Comment:
The version doesn't need to be changed in this PR. It will get updated automatically when publishing a new release of the theme.